src/settings.cpp

```diff
@@ -112,6 +112,7 @@ double res_scat_energy_min {0.01};
 double res_scat_energy_max {1000.0};
 vector<std::string> res_scat_nuclides;
 RunMode run_mode {RunMode::UNSET};
+SolverType solver_type {SolverType::MONTE_CARLO};
```

Comment:
All I'm saying is that random ray _is_ a Monte Carlo method. So there might be a better term out there. Not that it matters too much though. :D